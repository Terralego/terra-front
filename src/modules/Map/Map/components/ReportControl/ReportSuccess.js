import React from 'react';
import { Button } from '@blueprintjs/core';

const ReportSucess = ({ newReport, endReport, translate: t }) => (
  <div>
    <h1>{t('terralego.map.report_control.success.title')}</h1>
    <p>{t('terralego.map.report_control.success.message')}</p>
    <Button onClick={newReport}>{t('terralego.map.report_control.success.other_report')}</Button>
    <Button onClick={endReport}>{t('terralego.map.report_control.success.done')}</Button>
  </div>
);

export default ReportSucess;
