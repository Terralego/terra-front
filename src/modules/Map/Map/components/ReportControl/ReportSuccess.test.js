import React from 'react';
import renderer from 'react-test-renderer';

import ReportSuccess from './ReportSuccess';

it('should render correctly', () => {
  const tree = renderer.create(
    <ReportSuccess translate={m => m} />,
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
