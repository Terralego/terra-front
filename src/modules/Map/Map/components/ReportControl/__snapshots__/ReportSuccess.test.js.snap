// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<div>
  <h1>
    terralego.map.report_control.success.title
  </h1>
  <p>
    terralego.map.report_control.success.message
  </p>
  <button
    className="bp3-button"
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    type="button"
  >
    <span
      className="bp3-button-text"
    >
      terralego.map.report_control.success.other_report
    </span>
  </button>
  <button
    className="bp3-button"
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    type="button"
  >
    <span
      className="bp3-button-text"
    >
      terralego.map.report_control.success.done
    </span>
  </button>
</div>
`;
